node/db/Cargo.toml

```diff
@@ -8,16 +8,19 @@ edition     = "2021"
 repository  = "https://github.com/ChainSafe/forest"
 
 [features]
-default = ["snappy", "lz4", "zlib", "bzip2"]
+default = []
 
-snappy = ["rocksdb/snappy"]
-lz4    = ["rocksdb/lz4"]
-zlib   = ["rocksdb/zlib"]
-bzip2  = ["rocksdb/bzip2"]
+rocksdb = ["dep:rocksdb", "snappy", "lz4", "zlib", "bzip2"]
+snappy  = ["rocksdb/snappy"]
```

Comment:
Maybe it's better to use `dep:rocksdb?/snappy` syntax

Reply:
Can you elaborate on this? I am not really aware of this syntax

Reply:
> Starting in 1.60, you can add a ? as in "package-name?/feature-name" which will only enable the given feature if something else has enabled the optional dependency.

https://blog.rust-lang.org/2022/04/07/Rust-1.60.0.html
https://doc.rust-lang.org/cargo/reference/features.html#dependency-features

Reply:
@hanabi1224 do you mean `rocksdb?/snappy` then and not `dep:rocksdb?/snappy`? the `dep:` prefix makes `cargo` a bit angry.

Reply:
Yes! I guess ? Implies rocksdb is the dep instead of the feature